fio.run-file/README.md

```diff
@@ -0,0 +1,278 @@
+# fio example
+
+## Configuring to run the benchmark
+
+Included are the files required to run crucible.  They are:
+
+### fio.json
+
+This 'run-file' contains all of the necessary information to run a
+complex (multi-iteration, multi-sample) set of fio benchmarks on a
+'remotehost' endpoint (ie. a Linux box).
+
+### run.sh
+
+This file shows how a 'crucible run' command is used to run fio on a
+remotehost endpoint.  At the top of the file are a few values the user
+can change in order to direct where the test should be run and modify
+some aspects of it's behavior such as sample count and userenv (the
+userenv is the base Linux container that is used to build the run
+environment where fio will execute).
```

Comment:
hmm I'd just create a prepare-run.sh and leave the run to the 'crucible run ...' from cli.

Reply:
So I don't really explain it anywhere, but the other thing I'm trying to do here is show a very basic example of how a user can do complex configuration of their JSON file.  If you look at one of our other examples, [uperf](https://github.com/perftool-incubator/crucible-examples/tree/main/uperf), there is a lot of logic being used in the `run.sh` to configure the test.  We are going to want to use run-file in the same way but rather than building up the entire run invocation as a string the script is going to have to manipulate the JSON for the user.  This example is just the beginning of what we are going to be showing our users how to do.

Reply:
ack